.vscode/settings.json

```diff
@@ -0,0 +1,66 @@
+{
+    "extensions.ignoreRecommendations": true,
+    "extensions.showRecommendationsOnlyOnDemand": true,
+    "files.exclude": {
+        "**/*.a": true,
```

Comment:
Isn't there a lot of overlap with `.gitignore` here? any reason these can't be unified?